schema/compose-spec.json

```diff
@@ -581,12 +582,12 @@
       "items": {
         "type": "object",
         "properties": {
-          "capabilities": {"$ref": "#/definitions/list_of_strings"},
-          "count": {"type": ["string", "integer"]},
-          "device_ids": {"$ref": "#/definitions/list_of_strings"},
-          "driver":{"type": "string"},
-          "options":{"$ref": "#/definitions/list_or_dict"}
-        },
+            "capabilities": {"$ref": "#/definitions/list_of_strings"},
```

Comment:
are these whitespace changes intentional?

Reply:
They exist in the reference compose-spec.json file - maybe we should use a code formatter to prevent such issues